src/Eagle.ts

```diff
@@ -2640,11 +2646,25 @@ export class Eagle {
         if (triggerClass){
```

Comment:
This function performs the same behaviour (spinCollapseIcon) for a few different element types.

I found it a bit difficult to read and understand all the different cases.

I think triggerClass might be better named something like "isTranslationToggle".